packages/components/src/components/pagination/component.tsx

```diff
@@ -267,6 +267,7 @@ export class KolPagination implements API {
 	private getUnselectedPageButton(page: number): JSX.Element {
 		return (
 			<kol-button
+				aria-label={translate('kol-page-current', { placeholders: { page: String(page) } })}
```

Comment:
Laut originaler Ticket-Beschreibung ist das Attribut aber wohlmöglich gar nicht notwendig, nachdem ich auch das aria-label in L68 umgesetzt habe:

> In diesem Fall ist der Zusatz "Seite 1" nicht notwendig und die Ausgabe der Buttons kann bleiben wie sie gerade ist.

Hier bräuchte ich noch mal Feedback, bitte.